dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractMethodConfig.java

```diff
@@ -64,6 +64,19 @@
     // customized parameters
     protected Map<String, String> parameters;
 
+    /**
+     * forks for forking cluster
+     */
+    protected Integer forks;

```

Comment:
I think this is a new field? where do we use this value?

Reply:
Yep, new field.
It acts on the forkingcluster and specifies the opening of forks for concurrent access. U can see the issue for more details.
:)